functions/README.md

```diff
@@ -12,6 +12,15 @@ FIXME: explanation
 
     $ java -jar wort-0.1.0-standalone.jar [args]
 
+## Starting ring-server (inside folder containing project.clj)
+
+    $ lein ring server 3000
+
+## Building Sass
+
+    $ node-sass -wr ~/wort/functions/resources/public/css/scss \
```

Comment:
What is node-sass? How do I get it? Are we targeting a specific version?

Maybe something like [this lein tool](https://github.com/deraen/sass4clj) would make it easier?

Reply:
We can definitely do it through cloj instead. Let me see if I can get one of the popular ones added as a dep